doc/docs/master/enterprise/auth/enterprise-server/setup.md

```diff
@@ -20,8 +20,37 @@ The setup of an Enterprise Server requires to:
 ## 1 - Deploy An Enterprise Server
 
 ### Single-cluster deployment
-Deploying a Pachyderm cluster with the embedded enterprise server does not require any specific action.
-Proceed as usual:
+To deploying a Pachyderm cluster with the embedded enterprise server, an additional set of helm parameter values should be populated. 
+
+#### Create your values.yaml 
+Update your values.yaml with your enterprise license key, and auth configurations ([see example of values.yaml here](https://github.com/pachyderm/pachyderm/blob/master/etc/helm/examples/local-values.yaml)) or use our minimal example below.
+
+=== "values.yaml with activation of an enterprise license and authentication"
+
+	```yaml
+	pachd:
+	  	activateEnterprise: true
```

Comment:
Couple of things here:
- I think it would be good to add an explanation for all/most of these values to clarify what they do. 
- I think perhaps we should have a pachdHost which is the `hostname:port` or `ip:port` and that gets templated through everywhere `<PACH-IP>` is mentioned (instead of specifying multiple values)